### ISSUE TITLE
add a lobby map option: Repir all cam when game started

### ISSUE THREAD

**Issue by NetherDragonTw** [SubmergedAmongUs/Submerged#29]
**Is your feature request related to a problem? Please describe.**
in current version,when game started all cam will is fixed. i want to add a option in lobby computer settings.

**Describe the solution you'd like**
add a submerged settings tab icon, when host pressed, it will display submerged map settings, and have one option: 「Repir All Camera When Game is Started.」

**Additional context**
dev team pls added it to game!

**Comment by AlexejheroYTB** [comment]
I am going to offer a more in-depth explanation, as this is not the first time someone has requested this.

---

Previously, the cameras started broken at the beginning of the game, and crewmates had to fix them. We decided to change that and have them all working at the beginning for the following reasons:

- On an impostor-sided map like submerged, we shouldn't give crewmates a secondary objective of fixing cameras, instead impostors should have to go out of their way to disable the cameras if they want to.
- Most crewmates didn't care about cams. They ignored them and never actually used the security panel. Especially with the positioning of the cameras, some of them just aren't that good and most people decided to just ignore them.

By having all cameras enabled by default, this allows crewmates to have more information at the start of the game and it doesn't undermine the usability of the security panel.

 ---

Now, let's talk about adding this as an option.

Our aim when creating Submerged was to have as little as possible of a mod impact, allowing other mods to quickly and easily add Submerged compatibility. 

In order to add Submerged-specific game options, we would have to modify the following:

- the Game Options menu
- the Game Options UI list in the Lobby
- Game Options networking

Modifying these could very likely cause new conflicts with most other mods, given that they all add new options and modify all of these.

---

When releasing Submerged we decided that we would stay away from custom options, even removing the one option that was already there (Add Vitals Panel). We don't think it is worth _other mod developers'_ efforts to work around our custom options and as such we have no plans on adding any in the foreseeable future.
